board/views.py

```diff
@@ -35,13 +34,18 @@ class GetPosts(View):
 
     def get(self):
         """
+        Get the data from the database and send a JSON response.
+
         Get the `index` and `amount` from the query string params.
         Then, get the corresponding data from the database and send back a JSON response
         to the client.
 
         For example, if the user wished to load 30 more posts starting from the 51th post,
         the query string would be index=50&amount=30. The index is zero-based, inclusive at start
         and exclusive at the end.
+
+        Returns:
+            A JSON representation of the posts.
```

Comment:
Are you able to give a small description of what exactly one can expect from this JSON representation (like its structure or whatever)? Or has that not been coded yet?

Reply:
not coded yet 🙂

Reply:
that'll be done after DB